src/download.php

```diff
@@ -450,6 +518,43 @@
 
 
 
+if (isset($sFilter) && $sFilter == 'gene_public') {
+    // Ugly hack to run otherwise ignored filter of individualid on screenings.
+    // FIXME: make sure this filter is part of the query to get screenings from DB.
```

Comment:
The fixme simply states to find a better solution for the "ugly hack" in this block. The code would be easier to understand if the filters in this block are handled above in a similar way to the other filters.

I don't have a solution in mind, so it serves more as a reminder than a concrete point of action.